tutorials/server/install-and-configure-samba/install-and-configure-samba.md

```diff
@@ -0,0 +1,113 @@
+---
+id: install-and-configure-samba
+summary: Install and configure Samba to share files across a network.
+categories: server
+tags: tutorial,installation,ubuntu,file server,samba
+difficulty: 1
+status: Published
```

Comment:
missing *author*:
`author: Canonical Web Team <webteam@canonical.com>`

If you're not happy using your own address, you can use the above example.